util/slice/slice.go

```diff
@@ -3,7 +3,9 @@ package slice
 func RemoveString(slice []string, element string) []string {
```

Comment:
this method should either modify the argument (and not return anything) or not modify the argument - it should not do both